### ISSUE TITLE
No space between a mentioned name and toot text in the long aria-label for a mention

### ISSUE THREAD

**Issue by MarcoZehe** [nolanlawson/pinafore#1108]
When reading the aria-label for an article which contains a reply or mention, the text of the person being mentioned and the actual toot is being run together. There should be a space between the name and the beginning of the toot. Haven't checked if multiple mentioned names are run together as well, or if it's just from the last (or only) mentioned name to the actual toot text.

Probably related: A similar thing can be observed between a toot text and a link. The beginning of a link also follows the last character of a toot's text immediately, running the https together with the previous word, which makes it sound strange with a screen reader.

**Comment by nolanlawson** [comment]
Thanks for reporting the bug. You can check the option "Use short article ARIA labels" in the settings in the meantime as a workaround.

**Comment by nolanlawson** [comment]
Actually, I cannot reproduce this. For both replies and mentions, it appears that the `aria-label` correctly has a space after the mention.

For instance, if the toot text is:

> @admin1234560 @quux1234560 @baz1234560 hello

Then the `aria-label` is:

> foobar, \@admin1234560 @quux1234560 @baz1234560 hello, 2 minutes ago, @foobar1234560, Public

Maybe this is a browser thing, or an issue with a particular screen reader? I tested this in both Chrome and Firefox on Ubuntu and I'm seeing the same thing.

**Comment by nolanlawson** [comment]
Toot to reproduce the URL not having a space before it: https://metalhead.club/@thomas/101790153278327384
